unliftio/src/UnliftIO/Directory.hs

```diff
@@ -149,6 +152,20 @@ createDirectoryIfMissing :: MonadIO m => Bool -> FilePath -> m ()
 createDirectoryIfMissing create_parents path0 =
   liftIO (D.createDirectoryIfMissing create_parents path0)
 
+#if MIN_VERSION_directory(1,3,1)
+-- | Lifted 'D.createFileLink'.
+--
+-- @since 0.2.16.0
```

Comment:
Can you add a package.yaml version bump and a ChangeLog entry? Also, it's worth noting in the docs that this requires directory 1.3.1 or later.

Reply:
I hope I got everything right.

Reply:
Everything looks good to me, and CI is green. Thank you!